ui/src/tool-panel/location-track/dialog/location-track-edit-dialog.tsx

```diff
@@ -129,10 +125,20 @@ export const LocationTrackEditDialog: React.FC<LocationTrackDialogProps> = (
     const locationTrackOwners = useLoader(
         () =>
             getLocationTrackOwners().then((owners) =>
-                owners.sort((a, b) => a.name.localeCompare(b.name)),
-            ),
-        [],
+                owners.sort((a, b) => sortUnknownLocationTrackOwnerAsLast(a.name,b.name))
+            ),[],
     );
+
+    function sortUnknownLocationTrackOwnerAsLast(a: string, b: string) {
+        if (b === "Ei tiedossa") {
```

Comment:
Arvo "Ei tiedossa" tulee suoraan kannasta (location_track_owner.name), ja tuntuu hieman epäilyttävältä pitää sitä tässä kovakoodattuna. Olisiko parempaa ehdotusta?

Reply:
Tavallaan sama ongelma on siinäkin kun halutaan esivalita Väylävirasto.
Oisko meillä tarvetta lisäbiteille?

Reply:
Joo. Yksittäisenä ongelmana järjestyksenhän voi toki ratkaista niinkin että omistajiin laitetaan järjestysindeksi, mutta sekin tuntuu väärältä jos oikeasti halutaan vain "aakkosjärjejestys paitsi ei-tiedossa viimeisenä". Yleisesti komppaan Juusea - tässä tämä kyllä tuntuu nullilta joka on nyt arvona, mutta nuo muut vaihtoehdot voi muuttaa asiaa. Olkoon tässä näin ja palataan asiaan (toivottavasti pian) kun haluttu lopputulos on tiedossa.

Reply:
Mennään siis tällä, ja tehdään mahdollisesti isompia muutoksia sitten kun nuo epäselvyydet selkenee.